.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "bazil"]
```

Comment:
Delete this commit and the previous one (`temporarily remove bazil`) since we need relative-paths to avoid breaking recursive-github-cloning for https/ssh users.

Reply:
Good catch, done.